src/main/java/frc/robot/subsystems/Lights.java

```diff
@@ -1,24 +1,68 @@
 package frc.robot.subsystems;
 
+import com.ctre.phoenix.led.CANdle;
+import com.ctre.phoenix.led.CANdle.LEDStripType;
+import com.ctre.phoenix.led.CANdleConfiguration;
+import com.ctre.phoenix.led.RainbowAnimation;
 import edu.wpi.first.wpilibj2.command.SubsystemBase;
+import frc.robot.Constants;
+import frc.robot.RobotMap;
+import java.util.TreeMap;
 
 public class Lights extends SubsystemBase {
+  private TreeMap<LightCode, Integer[]> lightOptionsMap;
+  private CANdle candle = new CANdle(RobotMap.CANDLE_CAN_ID);
+  private CANdleConfiguration config = new CANdleConfiguration();
+  private LightCode currentLightStatus = LightCode.OFF;
+
   public enum LightCode {
     GAME_OBJECT, // green
     READY_TO_SCORE, // blue
     NO_TAG, // red
```

Comment:
I *think* NO_TAG should be what DEFAULT was before it was deleted (green larson animation), this is the same as piggie smalls